src/bindings/python/tests/test_graph/test_manager.py

```diff
@@ -67,9 +67,6 @@ def test_serialize_seperate_paths_kwargs(request, tmp_path):
     assert func.get_parameters() == res_model.get_parameters()
     assert func.get_ordered_ops() == res_model.get_ordered_ops()
 
-    os.remove(xml_path)
-    os.remove(bin_path)
```

Comment:
Files anyway will be removed by `pytest` because it's maintained by `tmp_path` fixture. The main reason of the change is because we can't remove IR before unmapping

Reply:
I decided to leave explicit delete because it may show any issues with `ov::Model` destruction. But it was required to move it to a scope different from `ov::Model` to guarantee it's destruction before files deletion